src/content-handlers/iiif/JQueryPlugins.ts

```diff
@@ -477,27 +483,33 @@ export default function jqueryPlugins($) {
       // Toggle function
       let expanded: boolean = false;
 
-      (<any>$self).toggle = function() {
-        $self.empty();
+      (<any>$value).toggle = function() {
+        $value.empty();
         const $toggleButton: JQuery = $('<a href="#" class="toggle"></a>');
+        let lessAriaLabel: string;
+        let moreAriaLabel: string;
         if (expanded) {
-          $self.html(expandedText + " ");
+          lessAriaLabel = [lessAriaLabelPrefix, labelText].join(' ');

```

Comment:
I think it would be better to insert a placeholder into the string, instead of using concatenation here. Otherwise, if we have languages that use different word orders than English, it will be harder to come up with effective translations. Take a look at the translation strings that use `{0}`, `{1}`, etc., to see how this works. (I can't remember exactly what you need to do to inject the values at the placeholders, but hopefully searching for some of those other strings in action will reveal the convention... if you have trouble finding it, let me know and I can dig deeper when time permits!

Reply:
Good idea, I've changed the 'prefixes' to 'templates' and implemented this. UV does the formatting with a custom util from @edsilv/utils.